trogdor/src/main/java/org/apache/kafka/trogdor/agent/AgentClient.java

```diff
@@ -199,10 +199,9 @@ private static void addWorkerIdArgument(ArgumentParser parser, String help) {
 
     public static void main(String[] args) throws Exception {
         ArgumentParser rootParser = ArgumentParsers
-            .newFor("trogdor-agent-client")
-            .build()
-            .defaultHelp(true)
-            .description("The Trogdor agent client.");
+                .newArgumentParser("trogdor-agent-client")
```

Comment:
There are many modifications since we upgraded argparse4j to 0.9 which brings us more effort to merge kafka upstream. Could you please search `.newFor()` and revert all the modifications?

<img width="1209" alt="image" src="https://github.com/AutoMQ/automq/assets/130526409/c9a39578-970c-447b-b2d6-847e6a7de9d7">

Reply:
Also please pay attention to the failed github action that may be helpful to find all the modifications. Thanks for your contributions.